ratis-client/src/main/java/org/apache/ratis/client/api/DataStreamOutput.java

```diff
@@ -25,6 +25,9 @@
 
 /** An asynchronous output stream supporting zero buffer copying. */
 public interface DataStreamOutput extends CloseAsync<DataStreamReply> {
+  /** Create a RaftClientRequest and send out it asynchronously */
+  CompletableFuture<DataStreamReply> streamRequestAsync();
```

Comment:
This method probably is not needed in this public user interface.  The request should be automatically sent but not be triggered by the user.